extensions/notebook/package.nls.json

```diff
@@ -13,6 +13,7 @@
 	"notebook.collapseBookItems.description": "Collapse Book items at root level in the Notebooks Viewlet",
 	"notebook.remoteBookDownloadTimeout.description": "Download timeout in milliseconds for GitHub books",
 	"notebook.pinnedNotebooks.description": "Notebooks that are pinned by the user for the current workspace",
+	"notebook.allowRoot.description": "Allow notebook to run as root",
```

Comment:
How about "Allow Jupyter server to run as root user" (or something to that effect)